snuba/utils/streams/topics.py

```diff
@@ -26,6 +26,7 @@ class Topic(Enum):
     QUERYLOG = "snuba-queries"
     PROFILES = "processed-profiles"
     DEAD_LETTER_QUEUE_INSERTS = "snuba-dead-letter-inserts"
+    DEAD_LETTER_TOPIC = "snuba-dead-letter-topic"
```

Comment:
I'm not sure if there will be a new dead letter topic or if `snuba-dead-letter-inserts` is the one we will be using. Whatever the topic name is will be updated here if it is going to be a new one.

Reply:
Are you saying everyone should use the same deal letter topic? How will we know which topic the messages originally came from in that case? I'm also not sure what `snuba-dead-letter-inserts` is on the line above.

Reply:
I believe the plan was always to have a single dead letter topic, with the original topic info encoded in the messages themselves. If you look at the example in the main comment of the PR you can see the structure I think we should use in this case. I think `snuba-dead-letter-inserts` is a topic that was created for @MeredithAnya's temporary DLQ.

Reply:
Update, each would use a different DLQ.